include/ListView/ListViewSmarty.php

```diff
@@ -139,10 +139,12 @@ public function process($file, $data, $htmlpublic)
 
         $contextMenuObjectsTypes = array();
         foreach ((array)$this->displayColumns as $name => $params) {
-            if (!isset($this->displayColumns[$name]['width'])) {
+            if (!isset($this->displayColumns[$name]['width']) || 0 === $adjustment) {
```

Comment:
Due to php "lazy evaluation" it would actually be better to put the: 
```php 
0 === $adjustment
```
on the left side for faster evaluation since it would cost less. More of a micro-optimisation and not a required change or anything, just thought I'd mention it :). Aside from that, change looks good to me, nice work and thanks for the contribution!